smac/facade/smac_ac_facade.py

```diff
@@ -449,6 +464,29 @@ def __init__(self,
                 type(intensifier)
             )
 
+        # initialize multi objective
+        # the multi_objective_algorithm_instance will be passed to the runhistory2epm object
+        multi_objective_algorithm_instance = None  # type: Optional[AbstractMultiObjectiveAlgorithm]
+
+        if scenario.multi_objectives is not None and num_obj > 1:  # type: ignore[attr-defined] # noqa F821
+            # define any defaults here
+            _multi_objective_kwargs = {"rng": rng, "num_obj": num_obj}
+            if multi_objective_kwargs is not None:
+                _multi_objective_kwargs.update(multi_objective_kwargs)
+            if multi_objective_algorithm is None:
+                for key, value in {
+                }.items():  # ParEGO default arguments  # TODO add hyperparameter
```

Comment:
We could already add the HPs for ParEGO here, otherwise it looks a bit confusing to me